.travis.yml

```diff
@@ -2,10 +2,18 @@ language: c++
 sudo: false
 env:
   - NODE_VERSION="0.10"
+  - NODE_VERSION="0.11"
   - NODE_VERSION="0.12"
+  - NODE_VERSION="iojs-v1"
```

Comment:
I'm not super strong on it, but I feel like we should test any version that we claim to support, just in case theres an incompatibilities.